[2022-05-04 13:05:09,898] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_dag.test_dag manual__2022-05-04T13:05:06.907149+00:00 [queued]>
[2022-05-04 13:05:09,947] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: test_dag.test_dag manual__2022-05-04T13:05:06.907149+00:00 [queued]>
[2022-05-04 13:05:09,948] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-05-04 13:05:09,950] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-05-04 13:05:09,951] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-05-04 13:05:09,988] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): test_dag> on 2022-05-04 13:05:06.907149+00:00
[2022-05-04 13:05:10,001] {standard_task_runner.py:52} INFO - Started process 1357 to run task
[2022-05-04 13:05:10,008] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'test_dag', 'manual__2022-05-04T13:05:06.907149+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpgg95oaq7', '--error-file', '/tmp/tmpyfle4lbs']
[2022-05-04 13:05:10,010] {standard_task_runner.py:77} INFO - Job 2: Subtask test_dag
[2022-05-04 13:05:10,203] {logging_mixin.py:109} INFO - Running <TaskInstance: test_dag.test_dag manual__2022-05-04T13:05:06.907149+00:00 [running]> on host 936fa37c1ac0
[2022-05-04 13:05:10,358] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=test_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-05-04T13:05:06.907149+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-04T13:05:06.907149+00:00
[2022-05-04 13:05:10,360] {logging_mixin.py:109} INFO - It's Working
[2022-05-04 13:05:10,362] {python.py:175} INFO - Done. Returned value was: None
[2022-05-04 13:05:10,392] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=test_dag, execution_date=20220504T130506, start_date=20220504T130509, end_date=20220504T130510
[2022-05-04 13:05:10,471] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-04 13:05:10,572] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
